### PR TITLE
DEVDOCS-4291 + DEVDOCS-4414: [update] Apps tutorial, updates per team incl node client source

### PR DESCRIPTION
**Pull request by slsriehl**
# [DEVDOCS-4291] and [DEVDOCS-4414]

## What changed?
* AppEx tutorial review notes
* Update code block titles
* Make GitHub links more accessible to screen readers

## Anything else?
* There's a google doc with requested changes attached to the ticket

[DEVDOCS-4291]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4291?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[DEVDOCS-4414]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4414?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by bc-traciporter — COMMENTED** [review]
good job @slsriehl that was a lot of work

File: docs/api-docs/apps/tutorials/sample-app-next/step-1-setup.md [hunk 1]
```diff
@@ -23,57 +23,59 @@ If you do not have Node.js installed, you can download it from [Node.js Download
...
 
-```shell
+```shell title="Install devDependencies"
 npm install --save-dev babel-plugin-styled-components @types/node @types/react typescript
 ```
 
 [babel-plugin-styled-components](https://www.npmjs.com/package/babel-plugin-styled-components) is a supplement to the styled-components library that, among other things, offers improved debugging and minification of styles.
 
-[@types/node](https://www.npmjs.com/package/@types/node) and [@types/react](https://www.npmjs.com/package/@types/react) contain TypeScript type definitions for Node.js and React.js respectively.
+[@types/node](https://www.npmjs.com/package/@types/node) and [@types/react](https://www.npmjs.com/package/@types/react) contain TypeScript type definitions for Nodejs and React.js respectively.
```
[the remark this answers is not in the document]
Comment (slsriehl) [reply docs/api-docs/apps/tutorials/sample-app-next/step-1-setup.md]: done locally with credit.

File: docs/api-docs/apps/tutorials/sample-app-next/step-1-setup.md [hunk 2]
```diff
@@ -23,57 +23,59 @@ If you do not have Node.js installed, you can download it from [Node.js Download
 ## Generate a package.json file
 
 <!-- theme: info -->
-> #### Note
-> * Make sure that you are using Node.js version 10.13+.
+> #### Nodejs version and custom dependencies
+> * Make sure that you are using Nodejs version 12.22+.
```
[the remark this answers is not in the document]
Comment (slsriehl) [reply docs/api-docs/apps/tutorials/sample-app-next/step-1-setup.md]: done locally with credit.

File: docs/api-docs/apps/tutorials/sample-app-next/step-1-setup.md [hunk 3]
```diff
@@ -23,57 +23,59 @@ If you do not have Node.js installed, you can download it from [Node.js Download
 ## Generate a package.json file
 
 <!-- theme: info -->
-> #### Note
-> * Make sure that you are using Node.js version 10.13+.
+> #### Nodejs version and custom dependencies
```
[the remark this answers is not in the document]
Comment (slsriehl) [reply docs/api-docs/apps/tutorials/sample-app-next/step-1-setup.md]: done locally with credit.

File: docs/api-docs/apps/tutorials/sample-app-next/step-1-setup.md [hunk 4]
```diff
@@ -6,13 +6,13 @@ Start by setting up your local development environment.
...
-For this tutorial, you need Node.js version 10.13+. To check your version of Node.js, run the following command in your terminal:
+For this tutorial, you need Nodejs version 12.22+. To check your version of Nodejs, run the following command in your terminal:
 
-```shell
+```shell title="Check Nodejs version"
 node -v
 ```
 
-If you do not have Node.js installed, you can download it from [Node.js Downloads](https://nodejs.org/en/download/). 
+If you do not have Nodejs installed, you can download it from [nodejs.org (downloads)](https://nodejs.org/en/download/).
```
[the remark this answers is not in the document]
Comment (slsriehl) [reply docs/api-docs/apps/tutorials/sample-app-next/step-1-setup.md]: done locally with credit.

File: docs/api-docs/apps/tutorials/sample-app-next/step-1-setup.md [hunk 5]
```diff
@@ -6,13 +6,13 @@ Start by setting up your local development environment.
 
 ## Install Node
 
-For this tutorial, you need Node.js version 10.13+. To check your version of Node.js, run the following command in your terminal:
+For this tutorial, you need Nodejs version 12.22+. To check your version of Nodejs, run the following command in your terminal:
```
[the remark this answers is not in the document]
Comment (slsriehl) [reply docs/api-docs/apps/tutorials/sample-app-next/step-1-setup.md]: done locally with credit.

File: docs/api-docs/apps/tutorials/sample-app-next/introduction.md [hunk 6]
```diff
@@ -19,7 +19,7 @@ At the end of this tutorial, you will have a functional single-click app that ca
 To successfully complete this tutorial, you need the following:
 
 * Experience using a text editor and a terminal app
-* Experience with [Node.js](https://nodejs.org/en/), [JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript), and [React](https://www.reactjs.org/)
+* Experience with [Nodejs](https://nodejs.org/en/), [JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript), and [React](https://www.reactjs.org/)
```
[the remark this answers is not in the document]
Comment (slsriehl) [reply docs/api-docs/apps/tutorials/sample-app-next/introduction.md]: done locally with credit.

File: docs/api-docs/apps/tutorials/sample-app-next/introduction.md [hunk 7]
```diff
@@ -2,7 +2,7 @@
 
 
 
-This tutorial demonstrates how to build and embed a BigCommerce app using Node.js, React, Next.js, and BigDesign, BigCommerce's library of React components. 
+This tutorial demonstrates how to build and embed a BigCommerce app using Nodejs, React, Next.js, and BigDesign, BigCommerce's library of React components.
```
[the remark this answers is not in the document]
Comment (slsriehl) [reply docs/api-docs/apps/tutorials/sample-app-next/introduction.md]: done locally with credit.

File: docs/api-docs/apps/apps-quick-start.md [hunk 8]
```diff
@@ -9,7 +9,7 @@ To successfully complete this tutorial, you will need the following:
 * [BigCommerce sandbox store](/api-docs/partner/getting-started/create-a-sandbox-store) (required to develop and test apps)
 * [Developer Portal Account](https://devtools.bigcommerce.com/) (required to register apps)
 * Experience using [npm](https://www.npmjs.com/)
-* Node.js 14.x
+* Nodejs 14.x
```
[the remark this answers is not in the document]
Comment (slsriehl) [reply docs/api-docs/apps/apps-quick-start.md]: done locally with credit.

File: docs/api-docs/apps/apps-quick-start.md [hunk 9]
```diff
@@ -1,6 +1,6 @@
 # BigCommerce Apps Quick Start
 
-In this quick start tutorial, you will create a [single-click app](/api-docs/getting-started/building-apps-bigcommerce/types-of-apps) using [Node.js](https://nodejs.org/en/), [React](https://www.reactjs.org/), [Next.js](https://nextjs.org/), and [BigDesign](https://developer.bigcommerce.com/big-design). 
+In this quick start tutorial, you will create a [single-click app](/api-docs/getting-started/building-apps-bigcommerce/types-of-apps) using [Nodejs](https://nodejs.org/en/), [React](https://www.reactjs.org/), [Next.js](https://nextjs.org/), and [BigDesign](https://developer.bigcommerce.com/big-design).
```
[the remark this answers is not in the document]
Comment (slsriehl) [reply docs/api-docs/apps/apps-quick-start.md]: done locally with credit.

File: docs/api-docs/apps/tutorials/sample-app-next/step-3-integrate-api.md [hunk 10]
```diff
@@ -12,29 +12,30 @@ This step demonstrates how to integrate the sample app with [Cloud Firestore](ht
...
 
-```shell
+```shell title="Install packages MySQL"
 npm install --save mysql jsonwebtoken swr
 ```
 
 <!-- theme: info -->
-> These instructions have been tested using the **firebase v8** package.  You can view a list of all the tested package versions in the [package.json file on the Step 3 branch](https://github.com/bigcommerce/sample-app-nodejs/blob/step-3-add-database/package.json) of this sample app's repo.
+> #### Firebase version
+> These instructions have been tested using the **firebase v9** package.  You can view a list of all the tested package versions in the [package.json file on the Step 3 branch](https://github.com/bigcommerce/sample-app-nodejs/blob/step-3-add-database/package.json) of this sample app's repo.
```
[the remark this answers is not in the document]
Comment (slsriehl) [reply docs/api-docs/apps/tutorials/sample-app-next/step-3-integrate-api.md]: done locally with credit.

File: docs/api-docs/apps/tutorials/sample-app-next/step-3-integrate-api.md [hunk 11]
```diff
@@ -153,38 +140,38 @@ You use a JSON Web Token (**JWT**) to securely transmit information encoded as a
...
 ## Update the auth lib page
 
 1. In the `lib` folder, open the `auth.ts` file.
 
 2. At the top of the file, add the following imports:
 
-```ts title="auth.ts" lineNumbers
+```ts title="Add imports auth.ts" lineNumbers
 import * as jwt from 'jsonwebtoken';
+import * as BigCommerce from 'node-bigcommerce'
```
[the remark this answers is not in the document]
Comment (slsriehl) [reply docs/api-docs/apps/tutorials/sample-app-next/step-3-integrate-api.md]: done locally with credit.

**Comment by slsriehl** [comment]
merging following review by @bc-traciporter.

**Comment by slsriehl** [comment]
Fixes #1383

**Pull request merged**